src/pages/index.js

```diff
@@ -105,27 +206,104 @@ function getCardElement(data) {
     cardLikeButton.classList.toggle("card__like-button_liked");
```

Comment:
```js

  cardLikeButton.addEventListener("click", () => {
    cardLikeButton.classList.toggle("card__like-button_liked");
  });
```

Please, delete the old code of `likes handling` because you have a new one in line 209